.ci/windows.groovy

```diff
@@ -407,13 +410,17 @@ def withBeatsEnv(boolean archive, Closure body) {
 def withBeatsEnvWin(Closure body) {
   final String chocoPath = 'C:\\ProgramData\\chocolatey\\bin'
   final String chocoPython3Path = 'C:\\Python38;C:\\Python38\\Scripts'
-  def goRoot = "${env.USERPROFILE}\\.gvm\\versions\\go${GO_VERSION}.windows.amd64"
+  // NOTE: to support Windows 7 32 bits the arch in the go context path is required.
+  def arch = is32bit() ? '386' : 'amd64'
+  def goRoot = "${env.USERPROFILE}\\.gvm\\versions\\go${GO_VERSION}.windows.${arch}"
 
   withEnv([
     "HOME=${env.WORKSPACE}",
+    "DEV_ARCH=${arch}",
+    "DEV_OS=windows",
     "GOPATH=${env.WORKSPACE}",
     "GOROOT=${goRoot}",
-    "PATH=${env.WORKSPACE}\\bin;${goRoot}\\bin;${chocoPath};${chocoPython3Path};${env.PATH}",
+    "PATH=${env.WORKSPACE}\\bin;${goRoot}\\bin;${chocoPath};${chocoPython3Path};C:\\tools\\mingw64\\bin;${env.PATH}",
```

Comment:
NOTE: this could be removed as soon as the packer images have been deployed with support to gcc.